.github/pull_request_template.md

```diff
@@ -1,19 +1,25 @@
-## Done
+### Done
 
-[List of work items including drive-bys]
+- [Add clear and detailed changes description]
 
-## Issue / Card
+---
```

Comment:
It's nice looking but I think we should keep the description as minimal as possible. The titles are already a good way to show a text septation.